djangosaml2/middleware.py

```diff
@@ -35,7 +35,7 @@ def process_response(self, request, response):
                 self.cookie_name,
                 path=settings.SESSION_COOKIE_PATH,
                 domain=settings.SESSION_COOKIE_DOMAIN,
-                samesite=None,
+                # samesite=None,
```

Comment:
comment? can we remove the commented code if not required?

Reply:
So the original djangosaml2 app had the "samsite" keyword argument but on testing, it was throwing an error regarding the same to be an unidentified keyword argument. I had commented this argument out in case it gets fixed in the future release of the djangosaml2 app. I will remove the keyword argument for now.